README.md

```diff
@@ -323,17 +324,17 @@ You can override this page (see [Pages](#pages)) to customize the HTML specially
 
 #### Rendering components
 
-The HtmldevBundle provides a `component` macro to render components from the styleguide anywhere in your application.
+*Rendering components through the `component` macro (`ui.component()`) is deprecated*
+
+To render components, simply use a Twig  include like below:
 
 This wil load `htmldev/components/cards/cover.html.twig` with the given properties:
  
 ```twig
-{% import 'ZichtHtmldevBundle:macros:components.html.twig' as ui %}
-
-{{ ui.component('cards/cover', { 
+{% include '@htmldev/components/cards/cover.html.twig' with {
```

Comment:
Ik zou graag alle templates op dezelfde plaats hebben.  Bijvoorbeeld in een 'components' directory in de Resources directory, waar ook de view templates staan.  Er is dan ook geen `@htmldev` twig config meer nodig